src/scene/shader-lib/chunks/standard/frag/gloss.js

```diff
@@ -1,13 +1,13 @@
 export default /* glsl */`
 #ifdef MAPFLOAT
-uniform float material_shininess;
+uniform float material_glossiness;
```

Comment:
considering you're already renaming the uniform .. should it be called material_gloss?

Reply:
Currently all shaders use 'glossiness' naming. I will create a ticket to address them all separately.